src/rgw/rgw_rest.h

```diff
@@ -161,6 +161,18 @@ class RGWGetObj_ObjStore : public RGWGetObj
   int get_params() override;
 };
 
+class RGWGetObjTags_ObjStore : public RGWGetObjTags {
+public:
+  RGWGetObjTags_ObjStore() {};
+  ~RGWGetObjTags_ObjStore() {};
```

Comment:
yeah I guess this should be a part of a cleanup of this entire file, which has a very similar pattern, maybe a seperate cleanup pr